tests/jenkins/TestDockerCopy.groovy

```diff
@@ -40,6 +40,7 @@ class TestDockerCopy extends BuildPipelineTest {
         super.setUp()
 
         // Variables
+        binding.setVariable('ALL_TAGS', false)
```

Comment:
Can you add a test for `All_TAGS` as `true` as well?

Reply:
That is already tested in the lib, basically the same outcome do we still need it here?

Reply:
If already covered in the lib repo then it is fine. This is just to verify the integrity of jenkinsfile i believe.

Reply:
Sure thanks.

Reply:
Please add for true too! If someone increments the version we want to make sure our jenkins file acts as expected.

Reply:
Also true is hazardous more than false so we want to make sure worst case scenario is covered as well

Reply:
Added now.